chroma-manager/tests/framework/integration/shared_storage_configuration/full_cluster/cluster_setup

```diff
@@ -10,6 +10,11 @@ CLUSTER_CONFIG=${CLUSTER_CONFIG:-"$(ls $PWD/shared_storage_configuration_cluster
 
 . $CHROMA_DIR/chroma-manager/tests/framework/utils/cluster_setup.sh
 
+# We need CLIENT_1 to exist after cluster_setup
+if [ -z ${CLIENT_1} ]l then
+    echo "Missing lustre_clients in cluster configuration"
+    exit 1
+fi
```

Comment:
`${CLIENT_1}` needs to be `""` quoted here.  If it is empty, it will be a syntax error.  But per below, I don't think adding this as a solution to proposed requirement [below](https://github.com/intel-hpdd/intel-manager-for-lustre/pull/474/files/14be42727b015aed13b9c647828d20850e656e37..1e351342c757fd8d23e3c859bde4959423de95ff#diff-4cbeadfa244f4bc2f8ecf2934b297766R41) is the right way to go.

I'm not sure that we want to force a client to be present in the configuration.  To be sure, there are probably places in the test suite that assume a client will be present but I would tend to feel that those are as erroneous as expecting one here.  There are tests that don't need a client and we wouldn't want to exclude running the whole test suite because of the lack of one.